### ISSUE TITLE
Feature: Dot in numbers + copy value

### ISSUE THREAD

**Issue by neohazuki** [Balvald/ArtemisScannerTracker#10]
Is possible in next version create dot in CR (money) and copy icon to export.
Is most simple to see the value and make calcs

**Comment by Balvald** [comment]
As I understood:

- Reformat the unsold value shown to have thousand seperators for readability (1,000,000 instead of 1000000)
- A button to copy value to clipboard. I'm assuming without the unit and thousands seperators. (with the plugin showing: "1,000,000 Cr." it'll put "1000000" onto the clipboard)

I'd make a case for not including the thousand seperators for the copy to clipboard button. (though its ambiguos to me if you've asked for this.)

And thats because of different languages that use different number formats and some do not automatically get converted by either google spreadsheets/LibreOffice Calc which depends on which languages basic number format it considers standard. (probably changeable through its respective settings but who the hell goes there for that?)

TL;DR: priorising either 1.000.000,00 or 1,000,000.00 is a pain for people that encounter their non-standard number format and there are a lot of number formats: https://en.wikipedia.org/wiki/Decimal_separator#Examples_of_use

@neohazuki sounds good, anything to add?

**Comment by Balvald** [comment]
currently working on all dev branches. will be closed once the 2-clonal-colony-range branch gets merged into main.
